### ISSUE TITLE
loss shooting to nan after restarting the model from last checkpoint

### ISSUE THREAD

**Issue by TheSeeker218** [NVIDIA/tacotron2#283]
![Screenshot (18)](https://user-images.githubusercontent.com/57669726/68774093-94196a80-0652-11ea-9290-47d15e06ce34.png)

Thanks in advance

**Comment by rafaelvalle** [comment]
Try reducing the learning rate or clipping the gradient norm.
